package.json

```diff
@@ -34,8 +34,7 @@
   "husky": {
     "hooks": {
       "commit-msg": "node tools/validate-commit-msg.js ${HUSKY_GIT_PARAMS}",
-      "pre-commit": "lint-staged && gulp scripts && gulp test",
```

Comment:
Sorry for misusing this PR, I'm testing out leaving comments for writing technical documentation in GitHub.